web-components/src/components/menu-overlay/scss/menu-overlay.scss

```diff
@@ -30,6 +30,11 @@
       &[data-show] {
         display: block;
       }
+
+      &[data-show-date-picker] {
+        display: block;
+        top: -15px !important;
```

Comment:
Will this work for responsive behaviour too? And can we do it without using important?

Reply:
changed this